ci/prow/e2e-incluster-build

```diff
@@ -12,21 +12,22 @@ oc label node ${NODE} task=kmm-ci
 echo "Deploy KMMO..."
 make deploy
 
-echo "Create a build secret..."
-oc create secret generic build-secret --from-literal=ci-build-secret=super-secret-value
+echo "Check that the kmm_ci_a module is not loaded on the node..."
+if oc debug node/${NODE} -- chroot host/ lsmod | grep kmm_ci_a; then
+ echo "Unexpected lsmod output - the module is present on the node before the module was applied to the cluster"
+ exit 1
+fi
 
 echo "Add a configmap that contain the kernel module build dockerfile..."
 oc apply -f ci/kmm-kmod-dockerfile.yaml
 
 echo "Add an kmm-ci Module that contains a valid mapping..."
-
-sed -e "s/KVER_CHANGEME/${KVER}/g" ci/module-kmm-ci-build.template.yaml | tee module-kmm-ci.yaml
-oc apply -f module-kmm-ci.yaml
+oc apply -f ci/module-kmm-ci-build.yaml
 
 # Wait for the build pod to be created. `kubectl wait` doesn't support such option,
 # see https://github.com/kubernetes/kubernetes/issues/83242.
 echo "Waiting for the build pod to be created..."
-timeout 5m bash -c 'until oc get pods -o json | jq -er ".items[].metadata.name | select(.? | match(\"build\"))"; do sleep 1; done'
+timeout 10m bash -c 'until oc get pods -o json | jq -er ".items[].metadata.name | select(.? | match(\"build\"))"; do sleep 1; done'
```

Comment:
Does it really take 10 minutes to create the build pod? If yes, we have a problem - it should be a matter of seconds.

Reply:
No. It should be triggered immediately (it is working locally but not in CI before timeout even after 10m for some reason. Still debugging this one.

Reply:
Seems like when I have increased it to 60m for debugging it passed (and failed with 10m) but I don't have timestamps on the logs so I don't know how long it actually took: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/rh-ecosystem-edge_kernel-module-management/343/pull-ci-rh-ecosystem-edge-kernel-module-management-main-e2e-in-cluster-build/1613426413178720256/artifacts/e2e-in-cluster-build/run-test/build-log.txt. I guess that it probably depends on the cluster we get?!